cairo/tests/utils/strategies.py

```diff
@@ -65,7 +81,8 @@
 uint64 = st.integers(min_value=0, max_value=2**64 - 1).map(U64)
 uint = uint64.map(Uint)
 uint128 = st.integers(min_value=0, max_value=2**128 - 1)
-felt = st.integers(min_value=0, max_value=DEFAULT_PRIME - 1)
+felt = st.integers(min_value=-DEFAULT_PRIME // 2, max_value=DEFAULT_PRIME // 2)
```

Comment:
this felt is always equal to default prime // 2, is it intentional?

Reply:
minus